File: Zombie_Killer/desktop/src/com/nopalsoft/zombiekiller/desktop/DesktopLauncher.java
package com.nopalsoft.zombiekiller.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.nopalsoft.zombiekiller.MainZombie;

public class DesktopLauncher {
    public static void main(String[] args) {
        LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
        config.width = 800;
        config.height = 480;
        new LwjglApplication(new MainZombie(), config);
    }
}

File: Zombie_Killer/desktop/src/com/nopalsoft/zombiekiller/desktop/DesktopLauncher.kt
package com.nopalsoft.zombiekiller.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.nopalsoft.zombiekiller.MainZombie

fun main() {
    val config = LwjglApplicationConfiguration()
    config.width = 800
    config.height = 480
    LwjglApplication(MainZombie(), config)
}
